features/google/src/model/importers/utils/importer-utils.js

```diff
@@ -0,0 +1,11 @@
+export function readCsvFromText(csvText) {
+    const rows = csvText.split("\n");
+    const headers = rows.shift().replaceAll('"', "").split(",");
+
+    const headersEnum = {};
+    headers.forEach((key, index) => (headersEnum[key] = index));
+    return {
```

Comment:
yeah the headersEnum is technically an Enum since enums don't exist in JS but instead are replaced by objects/hashmaps. In this case the point of the enum is to keep track of which column num the headers are in. Like this we can access the data we want just by referring to the header we want rather than some ambiguous index.

Reply:
Right, and that's kinda clear by the name, but still, you need to check out the source and implementation to find that out.

Reply:
So I guess a test that the correct headers are in the csv would make sense here